vagrant/Vagrantfile

```diff
@@ -107,8 +55,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         # Default value: false
         target.ssh.forward_agent = true
 
-        target.vm.provision "file", source: "./upgrade_packages.sh", destination: "upgrade_packages.sh"
-        target.vm.provision "file", source: "./blacklist_mongo.txt", destination: "blacklist_mongo.txt"
+        #target.vm.provision "file", source: "./upgrade_packages.sh", destination: "upgrade_packages.sh"
```

Comment:
This 2 lines will be removed with the on-build-config/vagrant/* when global team agree.